### PR TITLE
Update upgrade known issue about apply-manifest stuck

### PR DESCRIPTION
**Pull request by starbops**
Related issue: harvester/harvester#4484

### CODE REVIEW

File: docs/upgrade/v1-1-2-to-v1-2-0.md [hunk 1]
```diff
@@ -155,3 +155,60 @@ This could be caused by a misconfigured PDB. To check if that's the case, perfor
...
+
+1. Find out the terminating Pod's UID
+
+    ```
+    $ kubectl -n cattle-monitoring-system get pod prometheus-rancher-monitoring-prometheus-0 -o jsonpath='{.metadata.uid}'
+    33f43165-6faa-4648-927d-69097901471c
+    ```
+
+1. Get access to any node of the cluster via console or SSH
+1. Search for the related log messages in `/var/lib/rancher/rke2/agent/logs/kubelet.log` using the Pod's UID
```
Comment (LucasSaintarbor) [inline docs/upgrade/v1-1-2-to-v1-2-0.md:197]:
```suggestion
1. Search for the related log messages in `/var/lib/rancher/rke2/agent/logs/kubelet.log` using the pod's UID.
```

File: docs/upgrade/v1-1-2-to-v1-2-0.md [hunk 2]
```diff
@@ -155,3 +155,60 @@ This could be caused by a misconfigured PDB. To check if that's the case, perfor
...
+    ```
+
+    If kubelet keeps complaining about the volume unmount failure, you might want to apply the following workaround to make the upgrade proceed.
```
Comment (LucasSaintarbor) [inline docs/upgrade/v1-1-2-to-v1-2-0.md:203]:
```suggestion
     If kubelet continues to complain about the volume failing to unmount, apply the following workaround to allow the upgrade to proceed.
```

**Review by Vicente-Cheng — APPROVED** [review]
I am checking the whole page and with the lucas suggestion.
LGTM, thanks!

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**